mu4e/mu4e.texi

```diff
@@ -70,8 +70,8 @@ queries.
 @item Support for non-English languages (so ``angstrom''  matches ``Ångström'')
 @item Asynchronous: heavy actions don't block @t{emacs}@footnote{currently,
 the only exception to this is @emph{sending mail}; there are solutions
-for that though - see the @ref{FAQ}}
-@item Support for cryptography - signing, encrypting and decrypting
+for that though -- see the @ref{FAQ}}
+@item Support for cryptography -- signing, encrypting and decrypting
```

Comment:
Use `---`, the [emdash](http://www.thepunctuationguide.com/em-dash.html) (the [endash](http://www.thepunctuationguide.com/en-dash.html) "–" serves to denote ranges)